docs/user/openstack/observability.md

```diff
@@ -0,0 +1,349 @@
+# Observability of OpenShift on OpenStack
+
+This document explains how it is possible to correlate OpenStack and OpenShift
+metrics to have a better view of the stack and help troubleshoot issues
+affecting your clusters.
+
+## Make your OpenStack and OpenShift metrics available from the same metric store
+
+As a first step, you should setup a metric store instance to have access to both
+OpenStack and OpenShift metrics.
+
+There are a number of ways to get your metrics from the same instance. We'll
+document two of them: using Prometheus' remote-write to send the metrics to
+another instance, and using the federation endpoint to query metrics from
+a different instance.
+
+### Using remote-write to send RHOSO and OCP metrics to an external storage
+
+#### Set up the external storage
+
+In this example, we'll use an external storage to store the metrics. We'll
+assume that the external storage is another Prometheus instance.
+
+We will set up remote-write from both OpenStack and OpenShift, authenticating them with mTLS
+(mutual TLS). The target Prometheus needs to be configured to [accept client TLS
+certificates](https://prometheus.io/docs/prometheus/latest/configuration/https/),
+and [remote write](https://prometheus.io/docs/prometheus/latest/storage/#overview).
+
+<!--
+For testing purpose, we can do the following:
```

Comment:
I can't see these steps generated in the docs. I find it useful to have them listed
https://github.com/shiftstack/installer/blob/document_correlated_observability/docs/user/openstack/observability.md

Reply:
They are commented (with HTML comments (Markdown is a strict superset of HTML!)) because we target production setups in the docs, and this is just for testing. I think we will leave it there commented for the really curious.